### DISCUSSION TITLE
How to use grpc reflection

### DISCUSSION THREAD

**Discussion by gubtos** [AnWeber/httpyac#556] [General]
I would like to use grpc reflection to make requests to the services (like grpcurl). There is a form to do this?

**Comment by AnWeber** [comment]
Unfortunately, this is not currently supported, because I did not know it. But this looks interesting. It seems that in principle I could integrate it into my application using [grpc-reflection-js](https://www.npmjs.com/package/grpc-reflection-js) (few downloads, relatively new). I'll have to see what I can do, since I don't run a GRPC server myself or have access to one (maybe using [postman echo](https://learning.postman.com/docs/sending-requests/grpc/first-grpc-request/)). Maybe I have to create a docker that provides me with such an API.

**Reply by AnWeber** [reply]
grpc reflection is supported with version > v6.8
